pkg/metadata/host/host_windows.go

```diff
@@ -120,6 +121,8 @@ func getHostInfo() *InfoStat {
 	info.Procs = uint64(len(pids))
 	info.OS = runtime.GOOS
 
+	info.KernelArch = runtime.GOARCH
```

Comment:
Looks like this there's similar confusion in other places (example: https://github.com/DataDog/datadog-agent/blob/aba2d25fc63e365f85a779b69df8d0df5ef04530/pkg/metadata/host/host_windows.go#L91) but this will report the architecture for which the go binary was compiled, not the architecture of the kernel on which it's running (for example, I think this could report `386` on a 64-bit windows).

Just something to be aware of. You might want to leave this empty on Windows for now, to avoid having misleading information in the future.

Reply:
Good catch 👍  I'll leave this empty for now on Windows

Reply:
Actually @olivielpeau the Go documentation says:
> GOARCH is the running program's architecture *target*: one of 386, amd64, arm, s390x, and so on.
So even in the event cross-compilation, shouldn't this be correct?

Reply:
I was mostly thinking of the case where a program compiled for `386` would be running on an `amd64` system, which is a bit of an edge case. This is probably fine to leave as-is.